src/pages/components/ai-label/accessibility.mdx

```diff
@@ -0,0 +1,72 @@
+---
+title: AI label
+description: The AI label indicates an instance of AI in the UI.
+tabs: ['Usage', 'Style', 'Code', 'Accessibility']
+---
+
+<PageDescription>
+
+The AI label display and hide explainability information upon the click of the
+UI trigger element and can contain interactive elements.
+
+</PageDescription>
+
+<AnchorLinks>
+
+<AnchorLink>What Carbon provides</AnchorLink>
+<AnchorLink>Development considerations</AnchorLink>
+
+</AnchorLinks>
+
+## What Carbon provides
+
+Carbon already incorporates accessibility into the AI label component.
+
+### Keyboard interactions
+
+AI labels use the AI icon for the trigger button. These buttons are in the tab
+order an d are activated by pressing `Enter` or `Space`. The activation toggles
```

Comment:
Small Typo @aagonzales 
_These buttons are in the tab order_ **`and`** _are activated_